src/integrators/controllers.jl

```diff
@@ -23,6 +27,31 @@ reset_alg_dependent_opts!(controller::AbstractController, alg1, alg2) = nothing
 
 DiffEqBase.reinit!(integrator::ODEIntegrator, controller::AbstractController) = nothing
 
+# Standard integral (I) step size controller
+"""
+    NothingController()
+
+This Controller exists to match the interface when one does not want to use a controller,
+basically if you want to keep a fixed time step.
+"""
+struct NothingController <: AbstractController
+end
```

Comment:
I know this is bikeshedding but could there be a clearer name? It does not really do nothing but is the opposite of an adaptive controller. What about `StaticController`, `FixedController`, `NonadaptiveController`, or something like that? This would clarify the meaning for me a bit at least.

Reply:
Yes, you are right